src/minimega/router.go

```diff
@@ -341,6 +368,60 @@ func (r *Router) writeConfig(w io.Writer) error {
 	fmt.Fprintf(w, "bird routerid %v\n", r.routerID)
 	fmt.Fprintf(w, "bird commit\n")
 
+	// ***** firewall stuff ***** //
+
+	fmt.Fprintln(w, "fw flush") // no need to manage firewall state - just start over
+	fmt.Fprintf(w, "fw default %s\n", r.FW.defaultAction)
+
+	for name, chain := range r.FW.chains {
+		for _, rule := range chain.rules {
+			cmd := fmt.Sprintf("fw chain %s action %s", name, rule.action)
+
+			if rule.src != "" {
+				cmd = fmt.Sprintf("%s %s", cmd, rule.src)
+			}
+
+			cmd = fmt.Sprintf("%s %s %s", cmd, rule.dst, rule.proto)
+
+			fmt.Fprintln(w, cmd)
+		}
+
+		fmt.Fprintf(w, "fw chain %s default action %s\n", name, chain.defaultAction)
```

Comment:
Reject doesnt seem to work?
ERROR fw.go:187: defaulting FORWARD to REJECT exit status 1: iptables: Bad policy name. Run `dmesg' for more information.
@activeshadow

Reply:
@aherna this has been fixed... `REJECT` cannot be used as a default chain policy. To actively reject packets, the last rule in a chain needs to use `REJECT` as a target. As such, I've removed it as an option for the default firewall policy.